tailwind.config.js

```diff
@@ -2,7 +2,17 @@
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    fontFamily: {
+      'inter': ['Inter'],
```

Comment:
The design might not have all the fonts set correctly, but we don't actually need Inter so you can get rid of it